lib/YaST/Bootloader/BootloaderController.pm

```diff
@@ -10,7 +10,9 @@ package YaST::Bootloader::BootloaderController;
 use strict;
 use warnings;
 use YuiRestClient;
-use YaST::Bootloader::BootCodeOptionsPage;
+use YaST::Bootloader::BootCodeOptionsTab;
+use YaST::Bootloader::BootloaderOptionsTab;
+use YaST::Bootloader::BootLoaderSettingsPage;
```

Comment:
to know where we are with is_shown we need 3 different pages. In this case I think it is a good choice not to try 1-1 (1 controller - 1 page) because all the options in all tabs seems independent, it is just about organization/visualization in the screen. In other words, 1 controller 3 pages seems fine. But each page should contain the tab repeated and inheriting from NavigationBase. Having some page that represent the frame or the tab I'm not sure it could work well because you might not know where you are in each moment, repeating the tab in each page will give us the option to navigate only to the pages that make sense, not the current one for example. wdyt?

Reply:
It's just a logical problem for me: If I'm on the tab 1, and I want to switch to tab 2, where should I put the method to switch to tab 2 ? on Tab1Page, or on Tab2Page ? I can put it on Tab2Page, but that is somehow incorrect, as I am on tab 1 when I call this. To have a separate page representing the frame eliminates that logical issue. In this case, if we change settings within a tab, we will use a specific "is_shown" for it, so there is no ambiguity on where we are at that point, so we do not need to check it again when we press "next".

Reply:
I did as you suggest, In order to make sure the whole module is fully loaded, not just the tabs bar. Not resolving the conversation, so we have track of why we implemented it this way.